pkgs/development/python-modules/rtmixer/default.nix

```diff
@@ -0,0 +1,38 @@
+{ fetchFromGitHub
+, buildPythonPackage
+, cython
+, portaudio
+, cffi
+, pa-ringbuffer
+, sounddevice
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "rtmixer";
+  version = "0.1.1";
```

Comment:
```suggestion
  version = "0.1.1";
  disabled = isPy27;
```

Reply:
I've added the change you suggested, but I don't understand why it is necessary. Is `buildPythonPackage` sometimes overridden globally to an earlier Python version? I thought it would always point to the same (python 3.7 or something like that).

Reply:
The expressions in the package set are evaluated for all Python versions, thus writing the expression once gives you the possibility to build `python2.pkgs.rtmixer`, `python3.pkgs.rtmixer`, `python37.pkgs.rtmixer`, `pypy.pkgs.rtmixer`, etc.

Reply:
Ah, that makes sense. Thanks! :)